plugins/@grouparoo/calculated-property/public/templates/property/{{{id}}}.js.template

```diff
@@ -0,0 +1,23 @@
+exports.default = async function buildConfig() {
+  return [
+    {
+      id: {{{id}}},
+      name: {{{id}}},
+      class: "property",
+      sourceId: {{{sourceId}}}, // The ID of the Source that this Property belongs to - e.g. `sourceId: "calculated_property_source"`
+      type: "string", // The type of the Property.  Options are: {{{__typeOptions}}}
+      unique: false, // Will Profiles have unique records for this Property?
+      identifying: false, // Should we consider this property Identifying in the UI? Only one Property can be identifying.
+      isArray: false, // Is this an Array Property?
+      options: {
+        customFunction: customFunction // Write your customFunction below.  This will tell Grouparoo to execute it.
+      },
+      filters: null,
+    },
+  ];
+}
+
+function customFunction () {
```

Comment:
The function is written outside of the config object to allow for helpful, human-friendly formatting and is then parsed to a string in the template for the plugin to read.